scripts/slurm/run.bessemer.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Will need to increase the time for full benchmark runs.... 
+#SBATCH --time=8:00:00
+#SBATCH --partition=dcs-gpu
+#SBATCH --account=dcs-res
+# 1 GPU
+#SBATCH --gres=gpu:1
+# 1 CPU core - might want to increase this + memory to 1/4 of the node per GPU?
+#SBATCH --ntasks=1
+
+# Load modules (with matching gcc versions)
+module use /usr/local/modulefiles/staging/eb/all/
+module load CUDA/11.0.2-GCC-9.3.0
+# module load CMake/3.15.3-GCCcore-8.3.0
```

Comment:
Rmeove the commented out lines?

Reply:
@MILeach Can you check this please and commit a change if required.